python/seldon_core/batch_processor.py

```diff
@@ -649,12 +664,20 @@ def _send_batch_feedback(
 )
 @click.option(
     "--batch-size",
-    "-u",
+    "-s",
```

Comment:
why change of the flag here? wouldn't `-b` work better?

Reply:
batch-id was already using `-u`. Will that clash?

Reply:
Oh yeah, it would clash :D (only when I expended code above I noticed). 
I'd suggest `-b` then.

Reply:
Oh, `-b` also clash. Okay, then it's good.